myst_nb/transform.py

```diff
@@ -31,7 +31,12 @@
 }
 RENDER_PRIORITY["readthedocs"] = RENDER_PRIORITY["html"]
 RENDER_PRIORITY["singlehtml"] = RENDER_PRIORITY["html"]
+RENDER_PRIORITY["dirhtml"] = RENDER_PRIORITY["html"]
 RENDER_PRIORITY["linkcheck"] = RENDER_PRIORITY["html"]
+RENDER_PRIORITY["readthedocsdirhtml"] = RENDER_PRIORITY["html"]
+RENDER_PRIORITY["readthedocssinglehtml"] = RENDER_PRIORITY["html"]
+RENDER_PRIORITY["readthedocssinglehtmllocalmedia"] = RENDER_PRIORITY["html"]
+RENDER_PRIORITY["epub"] = RENDER_PRIORITY["html"]
```

Comment:
@choldgraf @mmcky can you think of any differences in priority between epub and html?

Note this is related to #179
Also I think probably at some point (maybe not in this PR) we just have a list of HTML aliases, rather than this dict copying

Reply:
None that I can think of, but I must admit I don't understand epub super well. I'd be fine with using the pattern we've already got here, and I agree w/ @chrisjsewell that we should just have a list of aliases rather than copying